DataFormats/TrackReco/interface/HitPattern.h

```diff
@@ -753,15 +779,33 @@ inline uint16_t HitPattern::getRPCregion(uint16_t pattern)
 
 ////////////////////////////// GEM
 inline uint16_t HitPattern::getGEMStation(uint16_t pattern)
+
 {
     uint16_t sss = getSubSubStructure(pattern), stat = sss >> 1;
     return stat + 1;
 }
+
 inline uint16_t HitPattern::getGEMLayer(uint16_t pattern)
 {
     return (getSubSubStructure(pattern) & 1) + 1;
 }
 
+////////////////////////////// ME0
+
+  // inline uint32_t HitPattern::getME0Roll(uint32_t pattern) {
+  //   uint32_t sss = getSubSubStructure(pattern), stat = sss >> 1;
+  //   return stat + 1;
+  // }
+  
+  inline uint32_t HitPattern::getME0Layer(uint32_t pattern) {
+    return (getSubSubStructure(pattern) & 1) ;
```

Comment:
this is not consistent with the format packed in the encodeMuonLayer (returns roll number).
If you don't know what to save, just not save anything (and maybe have no function either)